### ISSUE TITLE
Mean # of bounding boxes very low after 1.5K+ epochs

### ISSUE THREAD

**Issue by ahmedbaracat** [yhenon/keras-frcnn#25]
Hi,

Thank you @yhenon for your effort. I having a slight problem where I am training on VOC 2007 and 2012 datasets, but getting very low mean number of bounding boxes. The problem is apparent when I start testing on a mix of VOC2007 training images (which should be easy) and my own images: I don't get a single box detection. 

I hope you can help me out.

Example:
```
Epoch 1884:
Mean number of bounding boxes from RPN overlapping ground truth boxes: 10.5
loss_rpn_cls,0.77,loss_rpn_regr,0.11,loss_class_cls,1.49,loss_class_regr,0.36,class_acc,0.57,elapsed_time,9.77
```

**Comment by yhenon** [comment]
What settings are you using? Certainly it should have converged a bit more.

**Comment by yhenon** [comment]
I had pushed the code with an extremely low learning rate (1e-7) by mistake, sorry.
Here's my current output on pascal VOC:
```
Epoch 79:
Mean number of bounding boxes from RPN overlapping ground truth boxes: 22.6607843137
Classifier accuracy for bounding boxes from RPN: 0.83265625
Loss RPN classifier: 0.0798891891482
Loss RPN regression: 0.0895985876007
Loss Classifier classifier: 0.425928747375
Loss Classifier regression: 0.157889809495
```

**Comment by ahmedbaracat** [comment]
No worries :) I am currently the new one with 1e-5 learning rate. Hope this works...

Will close the issue if it does.

**Comment by yhenon** [comment]
I think maybe `1e-4` is better? Feel free to experiment.

**Comment by ahmedbaracat** [comment]
Will do. btw I noticed that the updated code is taking 100x time per epoch compared to the old version. Do you know what's the possible reason for that?

**Comment by yhenon** [comment]
I changed the number of iterations per epoch from 10 to 1000.

**Comment by ahmedbaracat** [comment]
Using the latest codebase solved the problem. I can now get reasonable testing predictions. Thank you
